### DISCUSSION TITLE
unit selection voices

### DISCUSSION THREAD

**Discussion by king-dahmanus** [RHVoice/RHVoice#575] [Ideas]
heylo there rhvoice developers, contributors and everyone else. I want to start a discussion around something that if it's applied, it could be a huge improvement. So, as we know, rhvoice uses statistical parametric synthesis with the help of hts. And it has something called mage which speeds it up right? So, what if there will be a new way to make fast natural sounding voices, a.k.a unit selection voices, with the same mage thing applied so the voices have the same degree of responsiveness as hts sp voices. Or, how about a whole new sp unit selection hybrid architecture with the flexibility, ease of training and responsiveness of statistical parametric synthesis with the clear naturalistic sounding of unit selection, maybe if more frequency details get added to the vocoder or however that subsystem works.  I'm trying to theorize an advancement in this thing because i believe it has a lot of potential. Thanks, and hope to see your replies.
p.s, mrs olga, hopefully this is better and doesn't result in the topic being closed like the previous issues. If there's still a mistake on my part, please tell me so it doesn't happen again

**Comment by king-dahmanus** [comment]
hello. I know that there are a lot of issues and prs so maybe this one slipped through some developers. Any comments on this theory?

**Comment by Olga-Yakovleva** [comment]
Unit selection is much more difficult to get right than spss. And yes, there actually are hybrid architectures, combining both approaches, which a pdeveloper would have to implement from scratch from a description in one of the papers on the subject. I myself am not planning to work on unit selection. But code contributions extending the core tts technology supported by RHVoice are welcome.

**Comment by king-dahmanus** [comment]
thanks for your input mrs. Olga. So you aren't deciding on it but you'll murge it if someone contributes it? Can you link to said paper please?
